### PR TITLE
Docs: MeshDataTool: showcase tool in code example

### PR DESCRIPTION
**Pull request by HaSa1002**
The code example needs some sort of enhancement, since you can't copy and run it and it does not show what you can do with the tool. I am however not so sure if this example is the best one, altough it demonstrates the tool pretty clearly.
grafik

### CODE REVIEW

File: doc/classes/MeshDataTool.xml [hunk 1]
```diff
@@ -7,16 +7,43 @@
...
 		    var vertex = mdt.get_vertex(i)
-		    ...
+		    # Your edits are going below. In this example we extend the mesh by one unit, which results in seperated faces as it is flat shaded.
+		    vertex += mdt.get_vertex_normal(i)
+		    # Save your change
 		    mdt.set_vertex(i, vertex)
 		mesh.surface_remove(0)
 		mdt.commit_to_surface(mesh)
-		[/codeblock]
+		var mi = MeshInstance.new()
```
Comment (akien-mga) [inline doc/classes/MeshDataTool.xml:24]: You forgot to assign the mesh?

File: doc/classes/MeshDataTool.xml [hunk 2]
```diff
@@ -7,16 +7,43 @@
...
+		[codeblocks]
+		[gdscript]
+		var mesh = ArrayMesh.new()
+		mesh.add_surface_from_arrays(Mesh.PRIMITIVE_TRIANGLES, CubeMesh.new().get_mesh_arrays())
 		var mdt = MeshDataTool.new()
 		mdt.create_from_surface(mesh, 0)
 		for i in range(mdt.get_vertex_count()):
 		    var vertex = mdt.get_vertex(i)
-		    ...
+		    # Your edits are going below. In this example we extend the mesh by one unit, which results in seperated faces as it is flat shaded.
```
Comment (akien-mga) [inline doc/classes/MeshDataTool.xml:18]:
```suggestion
		    # In this example we extend the mesh by one unit, which results in seperated faces as it is flat shaded.
```

This makes the sentence shorter to remove some horizontal scrolling, and I'd say it's understandable from the context that if you want to affect vertices, that's where you should do it.

**Review by akien-mga — APPROVED** [review]
Apart from a missing line in the GDScript codeblock, this seems good.

File: doc/classes/MeshDataTool.xml (hunk 1 above)
Comment (HaSa1002) [reply doc/classes/MeshDataTool.xml:24]: ups. fixed

**Comment by HaSa1002** [comment]
fixed

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**